src/types/PoqoSendReceiptReq.ts

```diff
@@ -0,0 +1,86 @@
+import { AppliedReceipt2 } from '../state-manager/state-manager-types'
+import { VectorBufferStream } from '../utils/serialization/VectorBufferStream'
+import { deserializeAppliedVote, serializeAppliedVote } from './AppliedVote'
+import {
+  deserializeConfirmOrChallengeMessage,
+  serializeConfirmOrChallengeMessage,
+} from './ConfirmOrChallengeMessage'
+import { deserializeSign, serializeSign } from './Sign'
+import { TypeIdentifierEnum } from './enum/TypeIdentifierEnum'
+
+export type PoqoSendReceiptReq = AppliedReceipt2
```

Comment:
The ser/de and type def for AppliedReceipt is already done, just use the existing function and avoid redundant code to prevent the same entity be having potentially different behavior.

Reply:
serialisation and deserialisation for appliedReceipt2 is incomplete in the file that you refer to and the type of serialisation is different as well. I don't want to touch that since it uses json stringify.

Reply:
We need enhancements and faster processing here for PoQO hence I wanted to give importance to the binary serialised code.

Reply:
> serialisation and deserialisation for appliedReceipt2 is not binary complete in the file that you refer to and the type of serialisation is different as well. I don't want to touch that since it uses json stringify.
Which I'm 100% aware. But this reasoning does not address my original review which is to avoid the situation two ser/de function for the same object type is implemented.  I see the need for perf fine tuning , we can still do this  while reducing this fallacy by moving serialization works to original appliedReceipt2 ser/de and reference it back to anywhere the appliedReceipt2 involved. 

I've made the changes to address this.

Reply:
thanks!